### ISSUE TITLE
The head navigator is not obvious

### ISSUE THREAD

**Issue by idcabana** [GameDevStudioUCSD/GDS_Website#9]
I think it would be better if the head navigator is always visible

**Comment by AnoXDD** [comment]
Yep that was what I was thinking. I'm open to this, but I'll make the header always visible and replace "HOME" with our GDS logo. Sounds good?

**Comment by idcabana** [comment]
that sounds fine for now
